### DISCUSSION TITLE
Dark mode is great but too contrasted

### DISCUSSION THREAD

**Discussion by sboulhazaiz** [community/community#23189] [Accessibility]
<p>The color is too black, it should be a bit more between black and grey like youtube or instagram (app) does for example !<br>
and it should have a shortcun to activate/reactivate it for those who do not use System dark mode (like windows users, windows dark mode is a bit botched)</p>

**Comment by gaknoll** [comment, marked as answer]
<p>Thanks <a class="mention" href="/sboulhazaiz">@sboulhazaiz</a> for taking the time to share the feedback - we very much appreciate it! We are exploring ways to make toggling between modes easier for our users. A keyboard shortcut is likely to be the first solution though we are also weighing additional opportunities (i.e. the profile settings dropdown). And regarding the feedback that the current dark mode is too black, we are also working on additional dark mode themes that either provide more contrast or less contrast to give our community more options for their preferred GitHub theme!</p>

**Comment by sboulhazaiz** [comment]
<p>Marvellous ! :smiley:<br>
That would be very cool to have a keyboard shortcut yes, why not !</p>
<p>Good luck ! :wink:</p>

**Comment by dieselburner** [comment]
<aside class="quote no-group" data-username="sboulhazaiz" data-post="1" data-topic="152869">
<div class="title">
<div class="quote-controls"></div>
 sboulhazaiz:</div>
<blockquote>
<p>The color is too black, it should be a bit more between black and grey like youtube or instagram (app) does for example !</p>
</blockquote>
</aside>
<p>Totally agree! Dark mode is too black IMO. Would love to have it dark gray! &lt;3</p>

**Comment by senderic** [comment]
<p>I agree - along with a Light / Dark mode, could we have a middle ground Gray or light-dark mode?</p>

**Comment by bdpiprava** [comment]
<p>I am little late the party but dark mode is not easy on eyes. It is great to know that team is coming up with different themes for dark mode :slight_smile:</p>

**Comment by harshakokel** [comment]
<p>I agree, the dark theme is too difficult on eye. Cannot see where the search box starts on home page.  Have to maximize screen brightness even to see the text “Find a repository”</p>
<p><img src="https://user-images.githubusercontent.com/858059/181075535-3ea37d36-1c63-40d2-9c92-1d2680de4543.png" alt="image" data-base62-sha1="hSYOIpcF5lCunq5r2fJq7PeBJ3m" width="335" height="157"></p>

**Comment by foloinfo** [comment]
<p>I agree light-grey will be nice.</p>
<p>I’ve been using <a href="https://chrome.google.com/webstore/detail/github-dark-theme/odkdlljoangmamjilkamahebpkgpeacp" rel="noopener nofollow ugc">GitHub Dark Theme chrome extension</a>  and I switched to official dark theme support, but I prefer this extension so I’m back to it.</p>
<p>If you are interested in the ‘light-grey’ kind of color scheme maybe you can try this extension until the official support arrives.</p>

**Comment by petedegraw** [comment]
Agreed with the original poster, and FWIW dark mode designs are mostly just a pain for those who can't use them. @github-staff Please let the user set "enable" dark mode instead of using "prefers-color-scheme" to set it automatically. People use dark themes in their browsers to see only the web content, and the browser fades out of the way. In the meantime, you must be logged into my GitHub account to have light mode. Using "prefers-color-scheme" is a bad pattern to follow because it doesn't work for all users.

**Comment by cehfisher** [comment]
Hi @sboulhazaiz - I wanted to follow up on this stale issue. Have the [five new dark theme options](https://github.com/settings/appearance) resolved this contrast issue for you? If so, could you please close the issue? If not, what additional changes are needed? Thank you! (cc @queenofcorgis)

**Comment by sboulhazaiz** [comment]
Sorryyyy I lost access to my github acc I still gotta renew my pw 😭 thank
you for closing it




Le mer. 29 mai 2024 à 17:21, Carie Fisher ***@***.***> a
écrit :
